sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/README.md

```diff
@@ -0,0 +1,17 @@
+﻿---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-service-bus
+name: Azure.Messaging.ServiceBus samples for .NET
+description: Samples for the Azure.Messaging.ServiceBus client library
+---
+
+# Azure.Messaging.ServiceBus Samples
+
+- [Send and receive messages from a queue](Sample01_HelloWorld.cs).
```

Comment:
We didn't mention about peek anywhere. What do you think if we include peek in this sample? 
something like: `Send, receive and peek messages from a queue`

Reply:
Good idea!

Reply:
Actually, I don't know that we want to advertise this as it isn't really a recommended usage.

Reply:
I was thinking we can add this only in sample.